gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -240,6 +243,12 @@ gst_tensor_trainer_class_init (GstTensorTrainerClass * klass)
           G_PARAM_READWRITE | GST_PARAM_MUTABLE_READY |
           G_PARAM_STATIC_STRINGS));
 
+  g_object_class_install_property (gobject_class, PROP_STOP_MODEL_TRAINING,
+      g_param_spec_boolean ("stop-model-training", "Stop model training",
+          "Stop model training being performed in a sub-plugin, "
+          "currently running epoch is finished to the end and the model is saved",
```

Comment:
"Set when the training is ready to be completed and saved. When it is set, the training session will be completed (stopped and saved) after the current epoch. This cannot be reverted."

Reply:
Thank you for your detailed comment.